public/app/core/components/sidemenu/BottomSection.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { SFC } from 'react';
+import _ from 'lodash';
+import SignIn from './SignIn';
+import BottomNavLinks from './BottomNavLinks';
+import { contextSrv } from 'app/core/services/context_srv';
+import config from '../../config';
+
+const BottomSection: SFC<any> = () => {
```

Comment:
Since we use airbnb style guide, it's better to use regular function:
```js
function BottomSection() {
```
https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless

Reply:
Sure, thought I had to do variables to do typed stateless functions. Fixed now.